code/modules/mob/living/simple_animal/abnormality/aleph/army_in_black.dm

```diff
@@ -0,0 +1,464 @@
+/mob/living/simple_animal/hostile/abnormality/army/Destroy()
+	UnregisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH)
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/army/Life()
+	..()
+	if(LAZYLEN(protected_targets))
+		for(var/mob/living/carbon/human/H in protected_targets)
+			if(!H.has_status_effect(STATUS_EFFECT_PROTECTION))
+				protected_targets -= H
+	return
+
+//no more nuzzling
+/mob/living/simple_animal/hostile/abnormality/army/AttackingTarget()
+	return FALSE
+
+/mob/living/simple_animal/hostile/abnormality/army/Move()
+	return FALSE
+
+/***Work procs***/
+//protect work grants you a buff in exchange for reducing its counter
+/mob/living/simple_animal/hostile/abnormality/army/AttemptWork(mob/living/carbon/human/user, work_type)
+	..()
+	if(work_type == "Protection")
+		if(datum_reference?.qliphoth_meter > 1)
+			if(user in protected_targets)
+				return FALSE
+			datum_reference.qliphoth_change(-1)
+			if((get_user_level(user)) < 4)
+				user.SanityLossEffect(TEMPERANCE_ATTRIBUTE)
+				return FALSE
+			protected_targets += user
+			user.apply_status_effect(STATUS_EFFECT_PROTECTION)
+			to_chat(user, "<span class='nicegreen'>You feel like you're in good company.</span>")
+			playsound(get_turf(user), 'sound/abnormalities/armyinblack/pink_heal.ogg', 50, 0, 2)
+		return FALSE
+	return TRUE
+
+//counter goes down when people die, repression/protection work is performed, or raised when an abnormality is suppressed.
+/mob/living/simple_animal/hostile/abnormality/army/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time)
+	if(work_type == ABNORMALITY_WORK_REPRESSION)
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/army/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(-1)
+	return
+
+//death counter from MoSB, it works exactly the same.
+/mob/living/simple_animal/hostile/abnormality/army/proc/OnMobDeath(datum/source, mob/living/died, gibbed)
+	SIGNAL_HANDLER
+	if(!IsContained())//Stops if it's breaching
+		return FALSE
+	if(died.z != z)//check for z-level congruency
+		return FALSE
+	if(isabnormalitymob(died))//counter restored if another abnormality dies
+		datum_reference.qliphoth_change(1)
+		return TRUE
+	if(!died.mind)//if it's not a player
+		return FALSE
+	if(!ishuman(died))//all simple mobs, ordeals, other npcs except for humans
+		return FALSE
+	death_counter += 1
+	if(death_counter >= 2)
+		death_counter = 0
+		datum_reference.qliphoth_change(-1)
+	return TRUE
+
+//*--Combat Mechanics--*
+/mob/living/simple_animal/hostile/abnormality/army/BreachEffect(mob/living/carbon/human/user)
+	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_ABNORMALITY_BREACH, src)
+	FearEffect()
+	Blackify()
+	SpawnAdds()//set its alpha to 0 and make it non-dense
+	for(var/mob/living/L in protected_targets)
+		L.remove_status_effect(STATUS_EFFECT_PROTECTION)
+	density = FALSE
+	alpha = 0
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
```

Comment:
It can still be targeted by mobs, I believe looking into invisibility code and variables made resolve this issue.

Reply:
It should probably also be made immortal via setting Brute damage_coeff to 0. You could actually do that in the base mob as Brute is ignored by records

Reply:
It never calls the parent and still has godmode. In tests it hasn't pulled aggro because of that

Reply:
This should probably be handled in a similar manner to Pinocchio, where the abnormality itself does not ever actually breach.